tools/docker/compose

```diff
@@ -91,6 +95,19 @@ case "$1" in
   dev)
     $dev ${@:2}
     ;;
+
+  eth:clean)
+    $base rm --stop --force devnet
```

Comment:
Do you think it'd be better to have the devnet node persist its data to a volume? Then we'd be able to stop the container and drop / backup the volume. This would give us the future capability of replaying block state for debugging by backing up the volume and restoring it repeatedly.

Reply:
I like the idea of using a volume - we might be able to reset the state without shutting down the container, which would be nice. I got kind of frustrated with that strategy because docker changes the ownership to root of everything in the mounted volume with gets annoying to work with :laughing:. It feels weird running sudo chown on the volume. Any thoughts on how to handle that?

Reply:
I believe you can run docker as a non-root user, maybe that would provide a potential solution?